docs/01-Using-Fleet/standard-query-library/standard-query-library.yml

```diff
@@ -516,3 +516,14 @@ spec:
   query: select * from shimcache
   purpose: Informational
   contributors: puffyCid
+---
+apiVersion: v1
+kind: query
+spec:
+  name: Get Mac and Linux machines with unencrypted primary disks.
+  platforms: macOS, Linux
+  description:
+  query: SELECT * FROM mounts m, disk_encryption d WHERE m.path= "/" AND m.device = d.name AND d.encrypted = 0;
+  purpose: Informational
+  contributors: DominusKelvin
+---
```

Comment:
```suggestion
```

```suggestion
```
> The below requested change is the same in #3137

I believe you have to remove the trailing `---` to pass the "Deploy Fleet Website" CI test. 

I think this is because the `---` signals the start of a new yaml document. You only need one at the top of the document (before the `apiVersion`)

Reply:
Fixed Noah!